x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/AuthorizationUtils.java

```diff
@@ -28,6 +28,7 @@
 import static org.elasticsearch.xpack.core.ClientHelper.ASYNC_SEARCH_ORIGIN;
 import static org.elasticsearch.xpack.core.ClientHelper.DEPRECATION_ORIGIN;
 import static org.elasticsearch.xpack.core.ClientHelper.ENRICH_ORIGIN;
+import static org.elasticsearch.xpack.core.ClientHelper.ENT_SEARCH_ENGINE_ORIGIN;
```

Comment:
could we make this just `ENT_SEARCH`? we might need it for more than just engines, e.g. analytics, connector related features

Reply:
That's a good point. Should we create multiple origins for that, or use a single one?